docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -56,8 +58,17 @@ Only the additional fields that the analysis calculated are written back, the
 ones that have been loaded in the loading data phase are not. The 
 {dfanalytics-job} matches the results with the data rows in the destination 
 index, merges them, and indexes them back to the destination index. When the 
-process is complete, the task is marked as completed and the {dfanalytics-job} 
-stops. Your data is ready to be evaluated.
+process is complete in case of an {oldetection} job, the task is marked as 
```

Comment:
I wonder if the last sentence (`the task is marked as completed...`) should move into the end of all the phases so we don't have to explain it twice. We could have a paragraph at the end that goes along the lines `Finally, after all phases are completed the task is marked as completed...`.